File: Assignment3/Program1.c
//1.Write a program which accept one number from user and print that number of even 
//numbers on screen.  
//Input :  7  
//Output:  2 4  6  8  10  12  14
#include<stdio.h>
void printEven(int);
int main()
{
    int iValue = 0;
    printf("Enter Number");
    scanf("%d",&iValue);
    printEven(iValue);
    return 0;
}
void printEven(int iNo)
{
    int iCnt = 0;
    if (iNo <= 0)
    {
        iNo = -iNo;
    }
    for(iCnt = 2; iCnt <= iNo*2; iCnt++)
    {
        if(iCnt % 2 == 0)
        {
            printf("%d ",iCnt);
        }
    }
}

File: Assignment33/Program1.java
// 1.Write a java program which accept number from user and return the count of even digits. 
// Input :  2395
// Output :  1 
// Input :  1018
// Output :  2 
// Input :  -1018
// Output :  2 
// Input :  8462
// Output :  4 

import java.util.*;

class Program1
{
	public static void main(String[] args) 
	{
		Scanner sobj = new Scanner(System.in);
		int iValue = 0,iRet = 0;

		System.out.println("Enter number");
		iValue = sobj.nextInt();

		Digit dobj = new Digit();
		iRet = dobj.CountEven(iValue);

		System.out.println("Count is : "+iRet);
	}
}

class Digit 
{  
	public int CountEven(int iNo)  
	{   
		int iDigit = 0,iCnt = 0;
		while(iNo != 0)
		{
			iDigit = iNo % 10;
			if(iDigit % 2 == 0)
			{
				iCnt++;
			}
			iNo = iNo / 10;
		}
		return iCnt;
  	} 
} 